src/components/App.js

```diff
@@ -118,7 +119,7 @@ const App = () => {
           </Route>
 
           <Route exact path='/cart/checkout'>
-            <Checkout />
+            <Checkout  />
```

Comment:
I see you have order, user, and component passed in as props in the component. If they are being used, add them in here also.

Reply:
Der. Totally forgot about this part. Thank you!